packages/fx-core/src/component/coordinator/index.ts

```diff
@@ -142,6 +142,7 @@ const AzureDeployActions = [
 const needTenantCheckActions = ["botAadApp/create", "aadApp/create", "botFramework/create"];
 
 class Coordinator {
+  @ErrorContextMW({ component: "Coordinator" })
```

Comment:
I'm wondering whether this could be designed as a class decorator to add component property to all methods of `Coordinator`?

Reply:
Class decorator  takes effect only when class is initialized.